using-mobile-extensions/mobile-core/README.md

```diff
@@ -1,34 +1,34 @@
 # Mobile Core
 
-The Mobile Core represents the core Adobe Experience Platform SDK that is required for every app implementation. The core contains a common set of functionality and frameworks, such as Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions. The following sections provide additional information about what is provided in the Mobile Core extension.
+Mobile Core is a mandatory Adobe Experience Platform SDK for app implementation. Mobile Core contains a common set of functionality and frameworks, including Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, and disk access routines. The following sections provide additional information about what is provided in the Mobile Core extension.
```

Comment:
Mobile Core is part of the SDK that is required for all app implementations. The other parts of the SDK are optional. Maybe say "The Mobile Core extension is mandatory for all Adobe Experience Platform app implementations"?